version.php

```diff
@@ -36,10 +36,10 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version    = 2016052401;
+$plugin->version    = 2016112701;
 $plugin->component  = 'mod_openmeetings';
 $plugin->maturity   = MATURITY_STABLE;
-$plugin->requires   = 2014051200;
-$plugin->release    = '2.0.0.4';
+$plugin->requires   = 2016112500;
```

Comment:
I see no changes why this plugin is no more compatible with older Moodle ....
Could you please explain it?